frontend/packages/console-shared/src/components/formik-fields/YAMLEditorField.scss

```diff
@@ -5,12 +5,7 @@
 
   &__editor {
     display: flex;
-    flex: 1 0 auto;
+    flex: 1 1 700px;
     flex-direction: column;
   }
-  &__sidebar {
-    display: flex;
-    flex: 1 0 0;
-    margin-left: 20px;
```

Comment:
This margin is needed somewhere otherwise it is inconsistent with the other pages with yaml and sidebar editor.

Reply:
yes, added back `margin-left`.